data_pipelines_cli/cli_utils.py

```diff
@@ -29,3 +30,11 @@ def get_argument_or_environment_variable(
         )
         sys.exit(1)
     return result
+
+
+def subprocess_run(args: List[str]):
+    try:
+        subprocess.run(args, check=True)
+    except subprocess.CalledProcessError:
+        echo_error(f"{args[0]} has run across an error")
+        sys.exit(1)
```

Comment:
This is a bit cluttered fix of CLI behaviour. Up to this point `subprocess.run` has been usually called without `check=True` flag, so ignoring non-zero exit codes. With the help of this helper function, non-zero exit codes will be stopping the CLI.